e2e_tests/tests/experiment/test_pytorch.py

```diff
@@ -7,116 +7,6 @@
 from tests import experiment as exp
 
 
-@pytest.mark.e2e_gpu
-@pytest.mark.parametrize("aggregation_frequency", [1, 4])
-@pytest.mark.parametrize("image_type", ["PT", "TF2"])
```

Comment:
last comment: are we replacing this two-image test functionality somewhere in the new tests?

Reply:
No, but we still have coverage of this in other e2e_tests (see `test_pytorch_cifar10_parallel` ). If we want to test the image, I would make a pytorch-image runner with a selected suite of tests.

Reply:
if we have the coverage in e2e under another test, I'm ok with that, and we can land this as-is.

Before we delete that _that_ test though, we need to bring that coverage into unit tests as well.